### ISSUE TITLE
Some entity embeddings are missing

### ISSUE THREAD

**Issue by yusanshi** [msnews/MIND#2]
In dataset MIND-small (downloaded from <https://msnews.github.io/>), it seems that at least the following entities from `news.tsv` is missing in `entity_embedding.vec` (in both training set and validation set).
```python
['Q1088533',
 'Q1101080',
 'Q12618419',
 'Q1268659',
 'Q1274953',
 'Q13479957',
 'Q1369657',
 'Q1433118',
 'Q14641401',
 'Q1518821',
 'Q15957025',
 'Q1618101',
 'Q16941365',
 'Q16971395',
 'Q16974590',
 'Q17005981',
 'Q17075762',
 'Q17157233',
 'Q1759320',
 'Q18206822',
 'Q1830767',
 'Q1974060',
 'Q1985787',
 'Q19876954',
 'Q2213191',
 'Q22909405',
 'Q22953685',
 'Q23016498',
 'Q24963455',
 'Q25098896',
 'Q2914964',
 'Q2920489',
 'Q3115816',
 'Q3309248',
 'Q3560381',
 'Q3642692',
 'Q3988',
 'Q43078954',
 'Q47195',
 'Q4870032',
 'Q48834885',
 'Q5161167',
 'Q5223107',
 'Q5255734',
 'Q5433004',
 'Q5504038',
 'Q55604490',
 'Q5571889',
 'Q55956059',
 'Q56248373',
 'Q56276162',
 'Q56315722',
 'Q5643122',
 'Q5956375',
 'Q59608476',
 'Q60745527',
 'Q60756198',
 'Q60767275',
 'Q6148929',
 'Q6322233',
 'Q6774029',
 'Q6815811',
 'Q6841333',
 'Q6972751',
 'Q6988097',
 'Q7061735',
 'Q7090844',
 'Q7098673',
 'Q7158944',
 'Q7233544',
 'Q724143',
 'Q7246726',
 'Q7250831',
 'Q7305326',
 'Q7316299',
 'Q7381085',
 'Q7389710',
 'Q7732041',
 'Q7784916',
 'Q7826008',
 'Q7845560',
 'Q7883884',
 'Q8023958',
 'Q8026356']
```

Take `Q1088533` for example, I opened the four files (`news.tsv`, `entity_embedding.vec` in training and validation set) and did a global search. But it was not found in `entity_embedding.vec`.
![image](https://user-images.githubusercontent.com/36265606/81844132-358d6480-9581-11ea-9ab8-89711866a7ce.png)

**Comment by yjw1029** [comment]
The problem is caused when creating the subgraph. Some entity node is isolated from other nodes so that they do not have entity embeddings.
